lib/medera.ex

```diff
@@ -39,4 +28,30 @@ defmodule Medera do
     Endpoint.config_change(changed, removed)
     :ok
   end
+
+  def child_specs(true) do
+    slack_children() ++ web_children() ++ minion_children()
```

Comment:
Note I mean OTP applications whereas your commented is on the child specs for the supervisor... there aren't any OTP children started here that aren't used.